client/stylesheets/_toCat.scss

```diff
@@ -5800,7 +5800,9 @@ textarea {
     display: none !important;
 }
 
-
+.homeCardSize b {
```

Comment:
Merge rule `.homeCardSize b` with rule on line 3203<br>Selector `homeCardSize` should be written in lowercase with hyphens